clang/test/3C/hash.c

```diff
@@ -1,7 +1,5 @@
 // RUN: 3c -alltypes %s | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
```

Comment:
Was this test not passing at all, before? I see you have not changed the expectation of what `-alltypes` will produce, in terms of changing the source file. So I'm confused how you are testing the change that you made to `ConstraintVariables.cpp`.

Reply:
This was one of the tests that I previously identified for not running clang on the output of 3c. It turns out, that would cause an error, explained in more detail with simplified example in issue #349. This change was verified by the addition of the compilation line, and the lack of effect on the suite of regression tests. I was hesitant to change the test in other ways, since I didn't write it or analyze why it exists. But I can add a "CHECKALL" line at the location that allowed it to compile.